commercetools/provider.go

```diff
@@ -55,34 +56,51 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("CTP_AUTH_URL", nil),
 				Description: "The authentication URL of the commercetools platform. https://docs.commercetools.com/http-api-authorization",
 			},
+			"mc_api_url": {
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("CTP_MC_API_URL", nil),
+				Description: "The API URL of the Merchant Center. https://docs.commercetools.com/custom-applications/main-concepts/api-gateway#hostnames",
+			},
 		},
 		ResourcesMap: map[string]*schema.Resource{
 			"commercetools_api_client":         resourceAPIClient(),
 			"commercetools_api_extension":      resourceAPIExtension(),
-			"commercetools_subscription":       resourceSubscription(),
-			"commercetools_project_settings":   resourceProjectSettings(),
-			"commercetools_type":               resourceType(),
 			"commercetools_channel":            resourceChannel(),
+			"commercetools_custom_application": resourceCustomApplication(),
```

Comment:
This is the new resource. The other changes here are because I sorted the list alphabetically.